src/googleclouddebugger/firebase_client.py

```diff
@@ -436,14 +454,15 @@ def _TransmitBreakpointUpdates(self):
           bp_ref.set(breakpoint_data)
 
           # Now strip potential snapshot data.
-          breakpoint_data.pop('evaluatedExpressions', None)
-          breakpoint_data.pop('stackFrames', None)
-          breakpoint_data.pop('variableTable', None)
+          summary_data = copy.deepcopy(breakpoint_data)
+          summary_data.pop('evaluatedExpressions', None)
+          summary_data.pop('stackFrames', None)
+          summary_data.pop('variableTable', None)
 
         # Then add it to the list of final breakpoints.
         bp_ref = firebase_admin.db.reference(
             f'cdbg/breakpoints/{self._debuggee_id}/final/{bp_id}')
-        bp_ref.set(breakpoint_data)
+        bp_ref.set(summary_data)
 
         native.LogInfo(f'Breakpoint {bp_id} update transmitted successfully')
 
```

Comment:
There's in except for socket.error below, should probably be removed now? It was a leftover from the gcp hub client version?

Reply:
I'll remove it for now; it was there to handle the errors raised by the cpu throttling in cloud functions.  It can easily be added back in if needed.